openmc/weight_windows.py

```diff
@@ -341,7 +370,7 @@ def from_xml_element(cls, elem, root):
         particle_type = get_text(elem, 'particle_type')
         survival_ratio = float(get_text(elem, 'survival_ratio'))
 
-        ww_shape = (len(e_bounds) - 1,) + mesh.dimension[::-1]
+        ww_shape = (len(e_bounds) - 1,) + tuple(mesh.dimension[::-1])
```

Comment:
should we instead insure `mesh.dimension` always returns a tuple since it is so similar to `shape` for `numpy.ndarray`?

Reply:
Good point. The rest of the `dimension` properties return a tuple. `RegularMesh` should too.